content/doc/book/pipeline/getting-started.adoc

```diff
@@ -1,7 +1,7 @@
 ---
 layout: section
+title: Getting Started with Pipeline
```

Comment:
The title in the front-matter for the book has been removed, that was erroneous. The `= First Header` is what is used as the document title, and should only be used, otherwise the PDF won't get the right title on sections